__tests__/app.test.js

```diff
@@ -65,6 +64,34 @@ describe('GET / api/reviews', () => {
         });
       });
   });
+
+test("2. 200: resolves with reviews sorted by date in descending order by default", () => {
+  return request(app)
+    .get('/api/reviews')
+    .expect(200)
+    .then(({ body }) => {
+      const { reviews } = body;
+      expect(reviews).toBeSortedBy('created_at',{descending:true})
+    });
+});
+
+test("3. 404: send a non-existent column to sort by", () => {
+  return request(app)
+    .get('/api/reviews?sort_by=vote; DROPTABLES')
+    .expect(404)
+    .then(({body:{msg}})=>{
+        expect(msg).toBe('No sort_by found');
+    });
+});
+test("4. 200: accept category query", () => {
+  return request(app)
+    .get('/api/reviews?category=euro game')
+    .expect(200)
+      .then(({body:{reviews}})=>{
+      expect(reviews).toHaveLength(1);
+        })
+});
```

Comment:
You can add more to this test, we can check if each review has the category given within it